kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -6,12 +6,30 @@ if [ $# -ne 3 ]; then
     exit 1
 fi
 
+cluster_name=`kubectl config get-clusters | grep -v NAME`
+
 ns=$1
 srv=$2
 conf=$3
 secretref=${4:-https://openstack.cern.ch:9311/v1/secrets/35145b52-18af-47a3-90d0-1861c51a9c65}
 
+if [[ "$cluster_name" == cmsweb-test[1-9]* ]] ; then
+   secretref="https://openstack.cern.ch:9311/v1/secrets/010bee01-f50f-40e6-b954-e24eae51d8d3"
+fi
+
+tmpDir=/tmp/$USER/sops
+
+# use tmp area to store sop binary
+if [ -d $tmpDir ]; then
+    rm -rf $tmpDir
+fi
+mkdir -p $tmpDir
+cd $tmpDir
+wget -O sops https://github.com/mozilla/sops/releases/download/v3.7.2/sops-v3.7.2.linux.amd64
+chmod u+x sops
```

Comment:
Not sure if we want to download the binary every time.
Maybe it's better to check if the binary is there and download only if it's missing.
We have anyways hardcoded the version of sops so I don't think this would create any issues.
What do you think?

Reply:
+1, we should check for existing executable first. May be also check if it provides `-version` or similar option

Reply:
I have adjusted script as per your suggestion. Thanks